src/CONST.js

```diff
@@ -125,6 +125,10 @@ const CONST = {
         STAGING: 'STG',
         PRODUCTION: 'PROD',
     },
+    DELAY: {
+        STARTUP: 8000,
```

Comment:
Please add comments to explain what these constants are for and why the values are what they are.